clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/StandardGceAttributeValidator.groovy

```diff
@@ -375,6 +386,46 @@ class StandardGceAttributeValidator {
     }
   }
 
+  def validateAutoscalingPolicy(GoogleScalingPolicy policy) {
+
+    validateNotEmpty(policy, "autoscalingPolicy")
+
+    if (policy) {
+
+      policy.with {
+        validateNonNegativeLong(minNumReplicas, "autoscalingPolicy.minNumReplicas")
+        validateNonNegativeLong(maxNumReplicas, "autoscalingPolicy.maxNumReplicas")
+        validateNonNegativeLong(coolDownPeriodSec, "autoscalingPolicy.coolDownPeriodSec")
+        validateMaxNotLessThanMin(minNumReplicas,
+          maxNumReplicas,
+          "autoscalingPolicy.minNumReplicas",
+          "autoscalingPolicy.maxNumReplicas")
+
+        customMetricUtilizations.eachWithIndex { utilization, index ->
+          def path = "autoscalingPolicy.customMetricUtilizations${index}"
```

Comment:
Index should probably be wrapped like this `[$index]` for readability in the case of an error.